tests/phpunit/Checker/Preparations/Force_Single_Plugin_Preparation_Tests.php

```diff
@@ -21,12 +21,12 @@ public function set_up() {
 		$this->plugin_basename_file = plugin_basename( WP_PLUGIN_CHECK_MAIN_FILE );
 	}
 
-	public function test_prepare_plugin_exists() {
+	public function test_prepare_plugin_does_not_exist() {
 
-		$preparation = new Force_Single_Plugin_Preparation( 'akismet/akismet.php' );
+		$preparation = new Force_Single_Plugin_Preparation( 'foo-plugin/foo-plugin.php' );
 
 		$this->expectException( 'Exception' );
-		$this->expectExceptionMessage( 'Invalid plugin akismet/akismet.php: Plugin file does not exist.' );
+		$this->expectExceptionMessage( 'Invalid plugin foo-plugin/foo-plugin.php: Plugin file does not exist.' );
 		$preparation->prepare();
```

Comment:
This test was inaccurate because in some environments like wp-env Akismet _does_ exist, so there wouldn't be an exception.

It's more robust to use a slug that definitely does not exist.